python/py-virtualfish/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem              1.0
 PortGroup               python 1.0
 PortGroup               github 1.0
 
-github.setup            adambrenecki virtualfish 2.5.4
+github.setup            justinmayer virtualfish 2.5.9
 # Change github.tarball_from to 'releases' or 'archive' next update
 github.tarball_from     tarball
```

Comment:
please act upon this comment; doing so will require you to update the checksums as well.